crates/starknet-server/src/api/http/endpoints/dump_load.rs

```diff
@@ -1,19 +1,54 @@
 use axum::{Extension, Json};
 
 use crate::api::http::error::HttpApiError;
-use crate::api::http::models::Path;
+use crate::api::http::models::{DumpPath, DumpResponse, LoadPath, LoadResponse};
 use crate::api::http::{HttpApiHandler, HttpApiResult};
 
 pub(crate) async fn dump(
-    Json(_path): Json<Path>,
-    Extension(_state): Extension<HttpApiHandler>,
-) -> HttpApiResult<()> {
-    Err(HttpApiError::GeneralError)
+    Json(path): Json<DumpPath>,
+    Extension(state): Extension<HttpApiHandler>,
+) -> HttpApiResult<Json<DumpResponse>> {
+    let starknet = state.api.starknet.write().await;
+    match path.path {
+        None => {
+            // path not present
+            starknet
+                .dump_transactions()
+                .map_err(|err| HttpApiError::DumpError { msg: err.to_string() })?;
+            Ok(Json(DumpResponse { path: "".to_string() }))
```

Comment:
Why are we returning an empty path to the user? I'm almost sure that's not what we did in devnet-py?

Reply:
right, in the old devnet we just return `Response(status=200)` so I fixed that to be `HttpApiResult<()>` in 2565cb00669394c78f6097a9bf0fd789f9b0bc68